nuxeo-core/nuxeo-core-storage-sql/nuxeo-core-storage-sql/src/main/java/org/nuxeo/ecm/core/storage/sql/jdbc/JDBCRowMapper.java

```diff
@@ -1151,9 +1152,9 @@ protected Serializable copyHier(Serializable id, Serializable parentId, String n
     /**
      * Gets the children ids and types of a node.
      */
-    protected List<IdWithTypes> getChildrenIdsWithTypes(Serializable id, boolean onlyComplex) throws SQLException {
+    protected List<IdWithTypes> getChildrenIdsWithTypes(Serializable id, boolean excludeRegularChildren, boolean excludeSpecialChildren) throws SQLException {
```

Comment:
Could you please switch the order of the arguments? The code you added for DBS uses `excludeSpecialChildren, excludeRegularChildren` everywhere.